README.md

```diff
@@ -168,6 +168,8 @@ Configurations that are supported:
   * For this configuration individual channel parameters can be configured. For example, if you define a `client.channel.params` 
 value with `key1=value1_#{topic},key2=value2_#{key}` in your configuration,
 and publish a message to "topic1" with key "key1", the `client.channel.params`  will be configured with `key1=value1_topic1,key2=value2_key1`
+*`message.name`
```

Comment:
For consistency with surrounding bullets:

```suggestion
* `message.name`
```

Reply:
Funnily, this looks fine on my local machine, I will commit your suggestion here neverthless.

Reply:
Oops, I'm getting "outdated suggestions cannot be applied"

Reply:
Looks like it's fixed in current state of this branch. 🤷 
https://raw.githubusercontent.com/ably/kafka-connect-ably/make_message_name_configurable/README.md